jdaviz/configs/specviz/plugins/unit_conversion/tests/test_unit_conversion.py

```diff
@@ -83,17 +51,15 @@ def test_spec_no_flux_no_uncert(specviz_app, spectrum1d):
     specviz_app.load_spectrum(spectrum1d, data_label=label)
 
     new_spectral_axis = "micron"
-
     spectrum1d.uncertainty = None
 
     conv_func = uc.UnitConversion.process_unit_conversion
     converted_spectrum = conv_func(specviz_app.app, spectrum=spectrum1d,
                                    new_spectral_axis=new_spectral_axis)
 
-    assert np.allclose(converted_spectrum.spectral_axis.value,
-                       RESULT_SPECTRAL_AXIS, atol=1e-5)
+    assert_quantity_allclose(converted_spectrum.spectral_axis,
```

Comment:
Instead of comparing `.value` and `.unit` separately, it is easier to use `assert_quantity_allclose`.